backend/controllers/auth.controller.ts

```diff
@@ -45,7 +45,7 @@ export const register = async (req: Request, res: Response) => {
       context: {
         subject: "Registration successfully",
         name: newUser.name,
-        link: `${BASE_URL}api/auth/verifyEmail/${emailVerificationToken}`,
+        link: `https://crm-webapp-zeta.vercel.app/api/auth/verifyEmail/${emailVerificationToken}`,
```

Comment:
Instead of using a hardcoded link, we can use environment variable so the link will not be dependent on a deployment. Currently we can use `FRONTEND_HOST`  environment variable which is defined in `backend/constants/index.ts`.

Reply:
But There's not `FRONTEND_HOST` in `.env.example` file. So you'd need to add it in the env on hosting platform.

Reply:
Great job on adding `FRONTEND_HOST` to the `.env.example` file! It's good practice to use this variable consistently throughout the codebase. Could you please update its usage here also?

**Additionally**, I noticed that in `.env.example`, the URL `https://crm-webapp-zeta.vercel.app/` is hardcoded. To make local development smoother, could you replace it with `http://localhost:3000`?